content/docs/coldchain/introduction.md

```diff
@@ -25,7 +25,7 @@ If a temperature breach happens, then a notification is displayed on every page
 
 Open mSupply integrates with the android based Cold Chain application; simply configure your Cold Chain application to sync with the Open mSupply server address and configure a username and password. Temperature sensors, breaches and logs will then appear within Open mSupply.
 
-<div class="note">The cold chain referred to here is the integration of cold chain features within Open mSupply. There is separate documentation for the android <a application href="/coldchain/introduction/">Cold Chain application</a>.</div>
+<div class="note">The cold chain referred to here is the integration of cold chain features within Open mSupply. There is separate documentation for the [Android Cold Chain application](/coldchain/introduction/)>
```

Comment:
```suggestion
<div class="note">The cold chain referred to here is the integration of cold chain features within Open mSupply. There is separate documentation for the <a href="/coldchain/introduction/">Android Cold Chain application</a>.</div>
```

Reply:
the reason for using the `<a>` tag is that if you use markdown in the div, you get this:

![image](https://github.com/msupply-foundation/msupply_docs/assets/9192912/5d5319de-b134-45e4-8dd7-e2f7ff7ffce2)

Reply:
mea culpa! Thanks @mark-prins